crates/pagecache/Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 edition = "2018"
 
 [features]
-default = []
+default = ["compression"]
```

Comment:
Do you think it could be better to enable it in the sled toml too? Even if it is already a default feature of pagecache? For clarity?